### PR TITLE
Math-Magicians MIlestone: Tests

### PR DESCRIPTION
**Pull request by CarlosIgreda**
For this Milestone I:

✅ Used GitFlow
✅ Used linters
✅ Set up a React App
✅ Used React documentation
✅ Used functional components
✅ Implemented the design only (JSX and CSS)
✅ Used React props
✅ Used React state
✅ Handled events in a React app
✅ Used React life cycle methods
✅ Understood the mechanism of lifting state up
✅ Used the correct React hook when fetching data
✅ Connected to an API within a functional component
✅ Showed different UI outputs depending on the component state
✅Used React Router
✅Used styles in a React app
✅Wrote unit tests with React Testing Library
✅Wrote unit tests with snapshots with Jest

### CODE REVIEW

**Comment by alexander16108** [comment]
Hi @CarlosIgreda , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that does not have a contribution from your coding partner

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by alexander16108** [comment]
Hi @CarlosIgreda , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no contribution from coding partner

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+describe('Home', () => {
+  it('should render correctly', () => {
+    const { container } = render(
+      <MemoryRouter>
+        <Home />
+      </MemoryRouter>,
+    );
+    expect(container).toMatchSnapshot();
+  });
```
Comment (Owusu-Desmond) [inline src/components/Home.test.js]: - [x] Please make sure that you fix this failing test that is the test for your home component :pray: 
  
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @CarlosIgreda, @emarkees  and @abiola-arobieke  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Home.test.js (hunk 1 above)
Comment (CarlosIgreda) [reply src/components/Home.test.js]: Hi @Owusu-Desmond, thanks for your feedback. Required changes are done.

image

**Review by franclobo — APPROVED** [review]
Hi team, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️
- Descriptive PR ✔️
- Correct Git flow ✔️
- All tests passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CarlosIgreda** [comment]
@franclobo, thank you so much for your feedback! 🚀

**Pull request merged**